index.html

```diff
@@ -11,35 +11,53 @@
 
 <p>The <a href="previous.html">Guides for Previous Versions</a> page has links to archives of older versions.</p>
 
-<p>The <a href="resources.html">Training Course Material</a> page contains links to material for printing out and using in training sessions, ZIP archives of the PDFs for all the current versions, customised versions of some of the Guides and other resources such as Omnigraffle&reg; and Word&reg; files.</p>
+<p>The <a href="resources.html">Training Course Material</a> page contains links to material for printing out and using in training sessions, ZIP archives of the PDFs for all the current versions, customized versions of some of the Guides and other resources such as Omnigraffle&reg; and Word&reg; files.</p>
 
-<p>The current release version of OMERO.insight is 5.2.7. You can tell what version of OMERO you are using by looking at the login screen, or selecting <span class="bold">About OMERO.insight</span> from the Help menu in OMERO.insight.</p>
+<p>The current release version of OMERO.insight is 5.3.0. You can tell what version of OMERO you are using by looking at the login screen, or selecting <span class="bold">About OMERO.insight</span> from the Help menu in OMERO.insight.</p>
 
 <p><img src="images/indexVersion.jpg" alt="OMERO.insight Version" style="width: 80%;"/></p>
 
-<p>All the Version 5 material on the User Help website applies to both version 5.1.4 and version 5.2, with the exception of the new features mentioned below, but note that version 5.1 is now unsupported and will not be updated.</p>
+<p>All the Version 5 material on the User Help website applies to version 5.2 and 5.3, with the exception of the new features mentioned below, but note that version 5.2 is now unsupported and will not be updated.</p>
```

Comment:
maybe use "maintenance" instead of unsupported
Since we might have to do a release if a "major" security issue is found